azure-pipelines.yml

```diff
@@ -0,0 +1,61 @@
+# C/C++ with GCC
+# Build your C/C++ project with GCC using make.
+# Add steps that publish test results, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/c-cpp/gcc
+
+trigger:
+- master
+
+pr:
+- master
+
+# File: azure-pipelines.yml
+
+jobs:
+- job: macOS
+  pool:
+    vmImage: 'macOS-10.13'
+  steps:
+  - script: |
+      brew update
+      brew install bison
+      brew install flex
+      brew install boost
+      brew install boost-python
+      brew install z3
+    displayName: 'package'
+  - script: |
+      export PATH="/usr/local/opt/bison/bin:$PATH"
+      export LDFLAGS="-L/usr/local/opt/bison/lib"
+      mkdir -p build && cd build
+      cmake ..
+      make
+      sudo make install
+      make test
+    displayName: 'build'
+
+- job: Linux
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - script: |
+      sudo apt-get update
+      sudo apt-get install bison
+      sudo apt-get install flex
+      sudo apt-get install libboost-all-dev
+      sudo apt-get install z3
+      sudo apt-get install libz3-dev
+    displayName: 'package'
+  - template: scripts/build.yml # Template reference
+
+- job: Windows
+  pool:
+    vmImage: 'vs2017-win2016'
+  steps:
+  - script: |
+      choco install winflexbison3
+      choco install python2
+      choco install boost-msvc-12
+    displayName: 'package'
+  - template: scripts/build.yml # Template reference
```

Comment:
Build and test without the ItSy interface in Windows to bypass Boost.Python dependency (temporary workaround).